static/docs/tutorials/index.md

```diff
@@ -0,0 +1,70 @@
+# DVC Basic Concepts and Features
```

Comment:
First level title should be about the chapter as a whole, not about this specific section that follows